README.md

```diff
@@ -106,9 +103,25 @@ var clickOutsideConfig = {
 };
 var EnhancedComponent = onClickOutside(MyComponent, clickOutsideConfig);
 ```
+or:
 
-Note that if you try to wrap a React component with a custom handler that the
-component does not implement, the HOC will throw an error at run-time.
+```js
```

Comment:
I would recommend changing the structure here, too. Rather than stating that there are two ways and then giving a list, I would advocate:
```
There are two ways. The first is to do X:

<<code here>>

The second is Y:

<<code here>>
```